### PR TITLE
Remove email from auth options

### PR DESCRIPTION
**Pull request by daniyarchambylov**
- Removed email address from auth params
- Bumped version to 2.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "armlet",
-  "version": "1.2.2-git",
+  "version": "2.0.0",
```
Comment (rocky) [inline package.json:3]: Le't use 2.0.0-git or something like this plays well with semantic versioning. Before the final release we'll drop the `-git`.

**Review by fgimenez — APPROVED** [review]
:+1: thanks!

File: package.json (hunk 1 above)
Comment (daniyarchambylov) [reply package.json:3]: Done

**Comment by rocky** [comment]
If it's good enough for @fgimenez , it is _fabulous_ for me. Thanks!

**Pull request merged**